lib/Auth/Process/COmanageDbClient.php

```diff
@@ -241,6 +241,9 @@ public function process(&$state)
                 $state['basicInfo'] = $basicInfo;
             }
             if (empty($basicInfo['id']) || empty($basicInfo['status']) || ($basicInfo['status'] !== 'A' && $basicInfo['status'] !== 'GP')) {
+                  if ($basicInfo['status'] === 'S') {
+                      throw new SimpleSAML_Error_Exception('You account is suspended. Please contact support for further assistance.');
```

Comment:
minor edit "Your account has been suspended"

Reply:
I have made the change.